### ISSUE TITLE
Toggle browser seems to be buggy

### ISSUE THREAD

**Issue by toondaey** [nvim-telescope/telescope-file-browser.nvim#304]
### Description

This normally toggles between viewing files/folders current directory and expanding all directories. However when I toggle to expand all directories, this is what I get.

<img width="1107" alt="Screenshot 2023-08-21 at 10 22 15" src="https://github.com/nvim-telescope/telescope-file-browser.nvim/assets/17554487/1ff1cd94-700f-4cf5-a5bd-60c22dc7189d">

### Neovim version

```markdown
NVIM v0.10.0-dev-1962+gd191bdf9d-Homebrew
Build type: Release
LuaJIT 2.1.0-beta3

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/homebrew/Cellar/neovim/HEAD-d191bdf/share/nvim"

Run :checkhealth for more info
```

### Operating system and version

macOs 13.4.1

### Steps to reproduce

1. Open the file browser
2. `<C-f>` in `Insert` mode or just `f` in `Normal` mode

### Expected behavior

It should show a list of folder paths starting from the cwd to the last folder for every path.

### Actual behavior

It just shows a blank like below.

<img width="1107" alt="Screenshot 2023-08-21 at 10 22 15" src="https://github.com/nvim-telescope/telescope-file-browser.nvim/assets/17554487/1ff1cd94-700f-4cf5-a5bd-60c22dc7189d">

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'
local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      {
        'nvim-telescope/telescope.nvim',
        requires = {
          'nvim-lua/plenary.nvim',
          'nvim-telescope/telescope-file-browser.nvim',
        },
      },
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
      display = { non_interactive = true },
    },
  }
end
_G.load_config = function()
  require('telescope').setup()
  require('telescope').load_extension('file_browser')
  -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end
if vim.fn.isdirectory(install_path) == 0 then
  print("Installing Telescope and dependencies.")
  vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end
load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]
```

**Comment by jamestrew** [comment]
Which commit are you on? Can you try updating `telescope-file-browser` and seeing if the issue still persists?

**Comment by toondaey** [comment]
I just updated and it looks to have been fixed with `ad7b637`. Thanks!

**Comment by jamestrew** [comment]
Good to hear - thanks.
